src/config.js

```diff
@@ -8,7 +8,18 @@ import {getLogo, macOsMenuBar} from './utils/logo'
 
 const isProduction = !isDev
 const currentURL = (name) => `file://${__dirname}/views/${name}.html`
-const ipfsPathFile = path.join(app.getPath('appData'), 'ipfs-electron-app-node-path')
+const ipfsAppData = (() => {
+  let p = path.join(app.getPath('appData'), 'ipfs-station')
+
+  if (!fs.existsSync(p)) {
+    fs.mkdirSync(p)
+  }
+
+  return p
+})()
+
+const ipfsPathFile = path.join(ipfsAppData, 'app-node-path')
+const ipfsFileHistoryFile = path.join(ipfsAppData, 'file-history.json')
 
 const ipfsPath = (() => {
```

Comment:
This function should use fs.existsSync instead of trying to catch for errors.

Reply:
done :)